tree/ntuple/v7/inc/ROOT/REntry.hxx

```diff
@@ -87,15 +87,15 @@ private:
 
    void AddValue(RFieldBase::RValue &&value)
    {
-      fFieldName2Token[value.GetField().GetFieldName()] = fValues.size();
```

Comment:
I think the comment of `fFieldName2Token` should be adjusted to clarify that the key may (also) be a qualified name of a subfield.